crypto/keys/keyring.go

```diff
@@ -501,6 +507,24 @@ func lkbToKeyringConfig(name, dir string, buf io.Reader, test bool) keyring.Conf
 	}
 }
 
+func newKWalletBackendKeyringConfig(name, _ string, _ io.Reader) keyring.Config {
+	return keyring.Config{
+		AllowedBackends: []keyring.BackendType{"kwallet"},
+		ServiceName:     "kdewallet",
+		KWalletAppID:    name,
```

Comment:
We should use a different value here it seems.

Reply:
This should be App's unique name